bin/inference/pycbc_inference_plot_movie

```diff
@@ -148,67 +164,70 @@ if len(opts.input_file) > 1:
 logging.info('Loading parameters')
 fp, parameters, labels, _ = io.results_from_cli(opts, load_samples=False)
 
-if opts.end_sample is None:
-    opts.end_sample = fp.niterations
+# get the total number of samples on disk
+thinned_by = fp.thinned_by
+nsamples = fp.niterations // thinned_by
+start_index = opts.start_index
+if start_index >= nsamples:
+    raise ValueError("given start-index {} is >= the number of samples in the "
+                     "input file {}".format(start_index, nsamples))
+end_index = opts.end_index
+if end_index is not None and end_index >= nsamples:
+    raise ValueError("given end-index {} is >= the number of samples in the "
+                     "input file {}".format(end_index, nsamples))
 
 if opts.log_steps and not opts.frame_number:
     raise ValueError("log-steps requires a non-zero frame-number to be "
                      "provided; see help for details.")
 
-# convert sample numbers to index
-start_index = opts.start_sample - 1
-end_index = opts.end_sample - 1
-
-# Define thin interval based on number of frames or frame step
-if opts.frame_number and opts.frame_step:
-    raise ValueError("Only one of frame-number or frame-step must be "
```

Comment:
@micamu I removed this because the `parser.add_mutually_exclusive_group` (line 115) takes care of enforcing that both are not provided, raising an error if not. This happens when the command line is parsed (inside `results_from_cli`), so it's impossible for you to get to this point if both options were provided. Here's what happens when you try to provide both arguments:
```
pycbc_inference_plot_movie --input-file H1L1V1-INFERENCE_150914_09H_50M_45UTC-5-1.hdf --parameters 'srcmass1:$m_1$' 'srcmass2:$m_2$' --plot-scatter --plot-marginal --z-arg snr --movie-file ~/WWW/scratch/prs/fix_plot_movie/test_frame_number.mp4 --output-prefix frames/frame --cleanup --frame-number 30 --verbose --frame-step 10
usage: pycbc_inference_plot_movie [-h] [--input-file FILE[:LABEL]
                                  [FILE[:LABEL] ...]] [-H]
                                  [--parameters PARAM[:LABEL] [PARAM[:LABEL]
                                  ...]] [--constraint CONSTRAINT[:FILE]
                                  [CONSTRAINT[:FILE] ...]] [--version]
                                  (--frame-number FRAME_NUMBER | --frame-step FRAME_STEP)
                                  [--start-index START_INDEX]
                                  [--end-index END_INDEX] [--log-steps]
                                  [--output-prefix OUTPUT_PREFIX] [--verbose]
                                  [--dpi DPI] [--nprocesses NPROCESSES]
                                  [--movie-file MOVIE_FILE] [--cleanup]
                                  [--plot-marginal]
                                  [--marginal-percentiles MARGINAL_PERCENTILES [MARGINAL_PERCENTILES ...]]
                                  [--plot-scatter] [--plot-density]
                                  [--plot-contours]
                                  [--contour-percentiles CONTOUR_PERCENTILES [CONTOUR_PERCENTILES ...]]
                                  [--mins PARAM:VAL [PARAM:VAL ...]]
                                  [--maxs PARAM:VAL [PARAM:VAL ...]]
                                  [--expected-parameters PARAM:VAL [PARAM:VAL ...]]
                                  [--expected-parameters-color EXPECTED_PARAMETERS_COLOR]
                                  [--plot-injection-parameters]
                                  [--z-arg Z_ARG] [--vmin VMIN] [--vmax VMAX]
                                  [--scatter-cmap SCATTER_CMAP]
                                  [--density-cmap DENSITY_CMAP]
                                  [--contour-color CONTOUR_COLOR]
                                  [--use-kombine-kde]
pycbc_inference_plot_movie: error: argument --frame-step: not allowed with argument --frame-number
```